docs/sdks/typescript/index.md

```diff
@@ -55,927 +67,740 @@ Each SpacetimeDB client depends on some bindings specific to your module. Create
 mkdir -p client/src/module_bindings
 spacetime generate --lang typescript \
     --out-dir client/src/module_bindings \
-    --project-path server
-```
-
-And now you will get the files for the `reducers` & `tables`:
-
-```bash
-quickstart-chat
-├── client
-│   ├── node_modules
-│   ├── public
-│   └── src
-|       └── module_bindings
-|           ├── add_reducer.ts
-|           ├── person.ts
-|           └── say_hello_reducer.ts
-└── server
-    └── src
+    --project-path PATH-TO-MODULE-DIRECTORY
 ```
 
 Import the `module_bindings` in your client's _main_ file:
 
 ```typescript
 import { SpacetimeDBClient, Identity } from '@clockworklabs/spacetimedb-sdk';
 
-import Person from './module_bindings/person';
```

Comment:
NIT: Sometimes you do this kind of import, but it's not that typical, and sometimes discouraged. More typical to do: 

```ts
import { DbConnection } from './module_bindings/index'
```

But if we do an import the variable we bind it to should follow TypeScript casing rules, so: `moduleBindings` or just `module`.

Reply:
> `import { DbConnection } from './module_bindings/index'`

We don't want to do the named import style of `import { stuff... } from path` because:
- We have to import a whole lot of stuff from the module bindings.
- In this document, we don't have a specific module that we're writing against, so we *can't* list all of the items that we'd need to import. E.g. what would the list of row types be?

> But if we do an import the variable we bind it to should follow TypeScript casing rules, so: `moduleBindings` or just `module`.

It seems weird to me to have the file name not match the imported variable name, and we've pretty firmly committed to using `module_bindings` as the directory name in all our TypeScript examples. But sure, if you think this is better.